scripts/ci

```diff
@@ -68,10 +75,11 @@ clean_cache() {
 }
 
 fail_clean() {
-    if [[ $TRAVIS_TEST_RESULT -ne 0 ]]; then
-        echo "Job failed. Cleaning cache..."
-        clean_cache
-    fi
+    #if [[ $TRAVIS_TEST_RESULT -ne 0 ]]; then
+    #    echo "Job failed. Cleaning cache..."
+    #    clean_cache
+    #fi
```

Comment:
Just remove this?

Reply:
I'll clean it up in a future revision :) I have another approach I want to take on CI in general that is much quicker and simpler.